light-curve/light_curve/light_curve_py/features/roms.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+from ._base import BaseSingleBandFeature
+
+
+class Roms(BaseSingleBandFeature):
+    def _eval_single_band(self, t, m, sigma=None):
+        n = len(m)
+        median = np.median(m)
+        return np.sum(np.aps(m - median) / sigma) / (n - 1)
```

Comment:
I think there is a typo here

```suggestion
        return np.sum(np.abs(m - median) / sigma) / (n - 1)
```